### DISCUSSION TITLE
How can I use a start-date in a milestone ?

### DISCUSSION THREAD

**Discussion by AndreHagemann** [community/community#102011] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hy,

I want to set a start date in a milestone. I only can set a due date. How can I do this ?

thx

André

**Comment by LuksJobs** [comment]
Hello Andre, 

GitHub does not have a built-in feature to set a start date for milestones. Milestones in GitHub are primarily used to track the progress of groups of issues or pull requests and typically include a due date to indicate when the milestone is expected to be completed.

As an alternative, you might consider using project boards to manage and track the progress of your work. Project boards allow you to create custom columns and move issues or pull requests between them. You can also add start and end dates to individual cards on a project board.
